src/styles/Location.css

```diff
@@ -0,0 +1,96 @@
+.location {
+  width: 100vw;
+  height: 100vh;
+  display: flex;
+  flex-direction: column;
+}
+
+#lottie {
+  width: 100vw;
+  height: 100vh;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.current-description {
+  background-color: rgba(255, 193, 193, 0.568);
+  display: flex;
+  border-radius: 0.5rem;
+  padding: 2rem;
+  margin: 4rem;
+  width: 500px;
+  box-shadow: 0 4px 8px 0 rgb(216, 114, 114), 0 6px 20px 0 rgb(209, 81, 81);
+  text-align: justify;
+}
+
+
+.history {
```

Comment:
I LOVE this, but you probably want to make sure it's responsive. I noticed we forgot to add it as a requirement this week, but that doesn't mean you shouldn't make everything you do responsive so see this as a reminder 😉

Reply:
<img width="491" alt="Skärmavbild 2024-04-23 kl  11 56 59" src="https://github.com/Technigo/project-labyrinth-zustand-vite/assets/51529201/902a4cac-ac8c-49ec-b36a-9f2ae4d39de6">